docs/source/markdown/podman-create.1.md

```diff
@@ -840,21 +840,27 @@ Specify the policy to select the seccomp profile. If set to *image*, Podman will
 
 Note that this feature is experimental and may change in the future.
 
-#### **\-\-secret**=*secret*
+#### **\-\-secret**=*secret*[**,**opt=opt]
 
 Give the container access to a secret. Can be specified multiple times.
 
 A secret is a blob of sensitive data which a container needs at runtime but
 should not be stored in the image or in source control, such as usernames and passwords,
 TLS certificates and keys, SSH keys or other important generic strings or binary content (up to 500 kb in size).
 
-Secrets are copied and mounted into the container when a container is created. If a secret is deleted using
-`podman secret rm`, the container will still have access to the secret. If a secret is deleted and
-another secret is created with the same name, the secret inside the container will not change; the old
+When secrets are specified as type `mount`, the secrets are copied and mounted into the container when a container is created.
+When secrets are specified as type `env`, the secret will be set as an enviornment variable.
+If a secret is deleted using `podman secret rm`, the container will still have access to the secret.
+If a secret is deleted and another secret is created with the same name, the secret inside the container will not change; the old
```

Comment:
Probably my misunderstanding, but where is the secret that is being deleted if it is not within the container?  I think more clarification is needed here as by reading this, it's not clear to me.  (Although it might be end of day fried brain here too.)

Reply:
It's being deleted from the secrets store, so, `podman secret create` creates a secret that is available for a container to use, and `podman secret rm` deletes it. This doesn't affect a secret inside a container if the container was created with that secret.

Reply:
A "for example" sentence or two explaining how a tls cert might be deleted, and then recreated on the host.  But does podman secret rm really delete the cert?

Reply:
Ah, the secret store is what I was missing and there's no mention of it in the man page here.  A quick sentence or two about that being the place secrets are stored created to and deleted from on the host would probably be good.

Reply:
I tried to clarify it a little in my latest push, let me know if its clearer